### ISSUE TITLE
Consolidate all data?

### ISSUE THREAD

**Issue by paul-bio** [bxlab/metaWRAP#318]
Hi guys. I just knew I could analysis my metagenome data(16samples) with metaWRAP. 
However in the first step, as posted in the homepage and several tutorials, the first thing I should do is 

cat  *_1.fastq > clean-read1.fastq
cat  *_2.fastq > clean-read_2.fastq
and then with those two 'clean-read1.fastq', 'clean-read2.fastq' files I can do assembly like this

metawrap assembly -1 clean-read1.fastq -2 clean-read_2.fastq -m 200 -t 96 --use-metaspades -o ASSEMBLY

I was wondering that how can I know which contigs are from which sample? In qiime2, there are metafile which I write detail information in csv file. Can I just run with 'clean-read1.fastq', 'clean-read2.fastq'?

+
my samples are to compare impact of typhoon in the ocean. 
I have 8 samples from one station(0m, 100m 200m .....1200m) before typhoon came and also 8 samples from same place but after typhoon.

I just added all *_1.fastq and *_2.fastq with clean-read1.fastq and clean-read2.fastq.

Anyway, thanks :)

**Comment by ursky** [comment]
I think your question is "how will I do my comparative analysis if I co-assemble all the samples together?". Your final co-assembly and subsequent bins/MAGs will indeed be a mixture of organisms from all samples. To perform your comparisons you will need to estimate the abundances of each contig and bin in each of your original samples (in the metawrap quant_bins module). From there you will see which of your contigs, bins, or even phyla (you will have all your contig taxonomies from the kraken module) went up or down in abundance after your typhoon. From there is more about what statistics you want to deploy to detect meaningfull differences from the noise - ill leave that up to you. 

Alternatively, you could process each sample seperately, but this has a major problem: each sample will only produce the most abundant organisms in that sample. So when comparing bin abundances between two samples, for example, you wont know if a sample truly did not have that bin or just barely below the assembly/binning threshold. By co-assembling you will be able to recover a much greater fraction of your community.

**Comment by paul-bio** [comment]
Thank you Ursky,
I truly understand. 

As I am new to metawrap and had experience qiime2, I was wonder how could metawrap identify which contig/bins are from which sample. In qiime, we make metadata that we could input categories. 

You mentioned  '_To perform your comparisons you will need to estimate the abundances of each contig and bin in each of your original samples (in the metawrap quant_bins module). '_   And this means in the heatmap, there will be sample ID on the x axis?

Your reply really helped me a lot. 
Thank you again URSKY.   :)

**Comment by ursky** [comment]
Correct. You will also have a text table of the abundance of each bin in each sample, so you will know which bins came from where.
